### DISCUSSION TITLE
How to get the list of wallet apps installed on the phone

### DISCUSSION THREAD

**Discussion by frankorbit** [WalletConnect/walletconnect-monorepo#1402] [Developer Support]
How to get the list of wallet apps installed on the phone.
Look at the picture
<img width="328" alt="截屏2022-08-31 下午6 03 34" src="https://user-images.githubusercontent.com/103110407/187653862-f0cb6c9b-3672-482b-8a6f-e2cce3d0841e.png">

**Comment by chadyj** [comment]
Take a look at the Cloud Explorer API for a full listing of wallets available.
https://docs.walletconnect.com/2.0/introduction/cloud-explorer

Getting a list of installed wallets on your phone depends on the OS, and may not be possible in many cases.

**Reply by frankorbit** [reply]
oh, thank you , copy that
